src/content-script/in-dapp-notifications/styles.module.css

```diff
@@ -0,0 +1,160 @@
+.hstack,
+.vstack {
+  display: grid;
+}
+
+.hstack {
+  grid-auto-flow: column;
+  grid-auto-columns: minmax(min-content, max-content);
+  align-items: center;
+  justify-content: initial;
+}
+
+.vstack {
+  grid-template-columns: minmax(0, auto);
+  align-items: start;
+}
+
+.notification {
+  position: absolute;
+  top: 64px;
+  right: 18px;
+  z-index: 9999;
+  display: flex;
+  align-items: center;
+  padding: 12px;
+  gap: 12px;
+  border-radius: 12px;
+  background: #fff;
+  color: #16161a;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
+  transition: transform 0.3s ease-in-out, opacity 0.3s ease-in-out;
+  transform: translateY(20px);
+  opacity: 0;
+  font-family: sans-serif;
+}
+
+.notification.show {
+  transform: translateY(0);
+  opacity: 1;
```

Comment:
Same question for opacity, too
https://github.com/zeriontech/zerion-wallet-extension/pull/516/files#r1691187373

Reply:
✅ cleaned up